src/main/java/emissary/pool/MoveSpool.java

```diff
@@ -31,7 +31,7 @@ public class MoveSpool implements Runnable {
     private static final Logger logger = LoggerFactory.getLogger(MoveSpool.class);
 
     // The payload FIFO
-    protected final LinkedList<SpoolItem> spool = new LinkedList<>();
+    protected final ArrayList<SpoolItem> spool = new ArrayList<>();
```

Comment:
Why use `ArrayList` here vs `ArrayDeque`? Performance isn't much of a concern based on possible sizes, so both are probably sufficient, but `ArrayDeque` has a nicer interface for operating on both ends and might be closer to the intent of the spool?

Reply:
> Why use `ArrayList` here vs `ArrayDeque`? Performance isn't much of a concern based on possible sizes, so both are probably sufficient, but `ArrayDeque` has a nicer interface for operating on both ends and might be closer to the intent of the spool?

Because these resolutions were auto-generated by errorprone.